js/lib/utils/abbreviateNumber.js

```diff
@@ -11,9 +11,9 @@
 export default function abbreviateNumber(number) {
   // TODO: translation
   if (number >= 1000000) {
-    return Math.floor(number / 1000000) + 'M';
+    return Math.floor(number / 1000000) + app.translator.trans('core.lib.number_suffix.mega_text');
```

Comment:
That's a good question. I guess we *could* apply `app.translator` to the whole thing...

But I get the feeling that the computer age is forcing its own syntax on the world, so expressions such as "1KB" end up getting used as-is regardless of how things were originally done in any given language. My guess is that we're safe enough leaving this one as a prefix for now.